### DISCUSSION TITLE
Username and Hostname Not Displaying

### DISCUSSION THREAD

**Discussion by wdpronovost** [starship/starship#4841] [Q&A] [answered]
I 100% expect that I did something incorrect.

Terminal: WARP
Shell: ZSH
System: MacOs (current)

Context: 
I had OHMYZSH installed and removed it. All was good. I had the user and host displaying on the ugly boring standard prompt.

I then installed Starship via Brew. Created a .zshrc file and put the `eval "$(starship init zsh)"` at the bottom of the file.

I got a nice pretty prompt of: 
> [directory] via [python] [python_version] >

That was everything I did. I then created the starship.toml file and added `format = "$all"` and no change. 

Any idea of what I am doing wrong? 

I also commented out everything but the starship init in my .zshrc file thinking maybe something in the file caused it but still no change. Happy to share the .zshrc if needed.

Thanks!

PS - I checked iTerm2 and no username or host name there either.

**Comment by matchai** [comment, marked as answer]
By default, the [`username` module](https://starship.rs/config/#username) is only shown when any of the following conditions are met:
- The current user is root/admin
- The current user isn't the same as the one that is logged in
- The user is currently connected as an SSH session
- The variable `show_always` is set to true

You can have the username always be shown with the following addition to your `starship.toml`:
```toml
[username]
show_always = true
```

Similarly, the [`hostname` module](https://starship.rs/config/#hostname) is only shown while SSHing.
It can be configured to always show with the following config addition:
```toml
[hostname]
ssh_only = false
```

**Reply by wdpronovost** [reply]
Thank you @matchai! I appreciate that and it will help me with going back to the docs and seeing what I missed.
